### DISCUSSION TITLE
What Visual Studio version for .Net 5.0 Release 10-Nov-2020

### DISCUSSION THREAD

**Discussion by roysurles** [dotnet/aspnetcore#27581] [General] [answered]
Hello, 

Which version of Visual Studio will we need?

16.7.7
16.8.0 Preview 6.0
Or will there be a new version update of Visual Studio next week?  Do we use the latest regular 2019 version or the latest preview 2019 version?

Thanks

**Comment by MagicAndre1981** [comment, marked as answer]
[Release channel was updated to 16.8](https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes#--visual-studio-2019-version-1680) (now 16.8.1), here you have official .net 5 support
